recipe/meta.yaml

```diff
@@ -131,26 +116,39 @@ outputs:
     requirements:
       build:
         - {{ compiler('c') }}
-        - make
+        - {{ stdlib('c') }}
+        - bison
+        - flex
+        - pkg-config
+      host:
+        - python {{ python }}
+        - meson
+        - yaml
+    files:
+      - include/fdt.h
+      - include/libfdt.h
+      - include/libfdt_env.h
+      - lib/libfdt.*
     test:
       commands:
         # tests/run_tests.sh -t libfdt depends on having dtc and the other binaries installed
         # we include those tests when packaging 'dtc' because it depends on having 'libfdt'
         - test -f $PREFIX/include/libfdt.h
         - test -f $PREFIX/include/fdt.h
         - test -f $PREFIX/include/libfdt_env.h
-        - test -f $PREFIX/lib/libfdt-{{ version }}${SHLIB_EXT}
         - test -f $PREFIX/lib/libfdt.a
         - test -f $PREFIX/lib/libfdt${SHLIB_EXT}
-        - test -f $PREFIX/lib/libfdt${SHLIB_EXT}.{{ major_ver }}
+        - test -f $PREFIX/lib/libfdt.{{ major_ver }}${SHLIB_EXT}  # [osx]
+        - test -f $PREFIX/lib/libfdt${SHLIB_EXT}.{{ major_ver }}  # [linux]
+        - test -f $PREFIX/lib/libfdt${SHLIB_EXT}.{{ version }}  # [linux]
```

Comment:
I'm assuming the switch to the more standard SOVER-handling on osx is due to the switch to meson. Given that `libfdt` advertises a minor-level pin in the run-export, and we're going from 1.6 to 1.7, this should be fine. For completeness, I check who uses `libfdt`, and it's actually only used by this feedstock itself.
```
$mamba repoquery whoneeds -c conda-forge -p linux-64 libfdt

Executing the query libfdt

conda-forge/noarch                                  17.4MB @   1.5MB/s 11.4s
conda-forge/linux-64                                40.0MB @   2.6MB/s 15.7s


 Name Version Build        Depends     Channel  Subdir
-------------------------------------------------------
 dtc  1.6.1   h166bdaf_0   conda-forge linux-64
 dtc  1.6.1   h166bdaf_1   conda-forge linux-64
 dtc  1.6.1   h166bdaf_2   conda-forge linux-64
```

Reply:
Thank you for verifying that. How did you check the run-export?

Reply:
See here
https://github.com/conda-forge/dtc-package-feedstock/blob/dff90d43a55dea85754ecca9472c7aa3094c58bf/recipe/meta.yaml#L129-L130